bip-csfs.md

```diff
@@ -0,0 +1,133 @@
+```
+BIP: XXX
+Layer: Consensus (soft fork)
+Title: CHECKSIGFROMSTACK
+Author: Brandon Black <freedom@reardencode.com>, Jeremy Rubin <j@rubin.io>
+Comments-URI: https://github.com/bitcoin/bips/wiki/Comments:BIP-XXXX
+Status: Draft
+Type: Standards Track
+Created: 2023-12-22
+License: BSD-3-CLAUSE
+```
+
+## Abstract
+
+This BIP describes two new opcodes for the purpose of checking cryptographic
+signatures in bitcoin scripts against data other than bitcoin transactions.
+
+## Summary
+
+When verifying taproot script spends having leaf version 0xc0 (as defined in
+[BIP 342]), we propose `OP_CHECKSIGFROMSTACK` to replace `OP_SUCCESS204`
+(0xcc).
+
+`OP_CHECKSIGFROMSTACKVERIFY` has semantics similar to `OP_CHECKSIG`, as
```

Comment:
This mentions OP_CSFSV, was that supposed to say just `OP_CHECKSIGFROMSTACK` or is this sentence obsolete?